gateway/models/runbooks.go

```diff
@@ -0,0 +1,163 @@
+package models
+
+import (
+	"database/sql"
+	"database/sql/driver"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"slices"
+	"time"
+
+	"github.com/hoophq/hoop/gateway/storagev2/types"
+	"github.com/lib/pq"
+	"gorm.io/gorm"
+	"gorm.io/gorm/clause"
+)
+
+type Runbooks struct {
+	ID                string                       `gorm:"column:id"`
+	OrgID             string                       `gorm:"column:org_id"`
+	RepositoryConfigs map[string]map[string]string `gorm:"column:repository_configs;serializer:json"`
```

Comment:
I recommend always using well-defined types in Go that serialize properly to JSON. Initially, I started with maps, but later learned how to implement this correctly using structured types. While proceeding with maps isn't necessarily problematic, using explicit types for models makes the code clearer and significantly improves API documentation.

Some examples:
- `./gateway/models/server_auth_config.go`
- `./gateway/models/datamasking.go`